THIRD_PARTY.md

```diff
@@ -5,5 +5,6 @@ Microsoft is working with multiple partners to develop protocols or an API to en
 
 - **[winget.pro](https://winget.pro)** offers private winget repository hosting on a paid basis.
 - **[WinGetty](https://wingetty.dev)** open source rest server for creating a private repository
+- **[winstall.app](https://winstall.app)** is updated every 15 minutes and offers an easy way to search for packages in the winget repository.
```

Comment:
This doesn't appear to be a third party repository (repository in this case is a place where winget packages are hosted), but instead is a third party tool for browsing winget packages. This should be in a separate section and have it's own header - something like `Third Party Tools`